sme_finance_application_schema/translations.py

```diff
@@ -41,18 +41,19 @@ def finance_application_v3_to_sme_v5(finance_application):
                   'date_finance_requested', 'finance_term_length', 'guarantor_available', 'purpose'):
         if field in finance_application['finance_need']:
             sme_v5[field] = finance_application['finance_need'][field]
+
     if finance_application.get('actors'):
-        directors = [x for x in finance_application['actors'] if x['role'] == 'director']
-        for director in directors:
-            if 'value_of_property_equity' in director:
+        director_or_guarantors = [x for x in finance_application['actors'] if x['role'] in ('director','guarantor')]
+        for actor in director_or_guarantors:
+            if 'value_of_property_equity' in actor:
                 sme_v5.setdefault('directors_houses', 0)
-                sme_v5['directors_houses'] += director['value_of_property_equity']
+                sme_v5['directors_houses'] += actor['value_of_property_equity']
```

Comment:
you're mapping guarantors data to director data here (and below)

Reply:
Yeah, I realise this. I think the original translation was incorrect but it didn't matter because we only ever took Director information in finance_application_v3.

The reason I believe it should be both Director and Guarantor even though sme_v5 only says "Director" is because the concept of a separate Guarantor didn't exist when we did that. We split them in actor_v1 but sme_v5 always implied that Guarantors were Directors and visa versa.

I think actually it's more correct to do the directors_houses, pensions, etc only with Guarantors as these are for looking at "assets used to secure the loan" and some Directors may not wish to put their house on the line! However, since we can't support multiple Roles for actors yet...